piglow/examples/strobe.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"time"
+
+	"github.com/goiot/devices/piglow"
+	"golang.org/x/exp/io/i2c"
+)
+
+func main() {
+	p, err := piglow.Open(&i2c.Devfs{Dev: "/dev/i2c-1", Addr: 0x54})
+	if err != nil {
+		panic(err)
+	}
+
+	defer p.Close()
+
+	if err := p.Reset(); err != nil {
+		panic(err)
+	}
+
+	for {
```

Comment:
I just wrote a little test program and if you control+c a program, it will _not_ execute the `defer`.
```go
package main

import (
	"fmt"
	"time"
)

func main() {
	defer fmt.Println("closing...")

	for {
		fmt.Print(".")
		time.Sleep(100 * time.Millisecond)
	}
}
```
```sh
$ go run main.go
.....................^Csignal: interrupt
```
I'll add the trap to the program.